src/base/system_parameter.h

```diff
@@ -463,6 +463,8 @@ enum param_id
   PRM_ID_ER_LOG_READ_LOG_PAGE,
   PRM_ID_ER_LOG_READ_DATA_PAGE,
 
+  PRM_ID_LOG_CALC_REPL_DELAY,
```

Comment:
To be in sync with the other debug logging parameters.
```suggestion
  PRM_ID_ER_LOG_CALC_REPL_DELAY,
```

Reply:
Done. I thought 'er' was for 'error' and this one is not about any error.